### PR TITLE
added ibc-rate-limiting module for sdk 47

### PR DESCRIPTION
**Pull request by sampocs**
Added the [v1.0.1](https://github.com/Stride-Labs/ibc-rate-limiting/releases/tag/v1.0.1) rate limiting release to ibc v7

### CODE REVIEW

**Comment by Reecepbcups** [comment]
lint issue is probably a version mismatch between the github ci & your local version. So make sure to update either or to latest. usually fixes

File: modules/rate-limiting/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright for interchainquery public infrastructure attributed to 2022 Quicksilver
```
Comment (Reecepbcups) [inline modules/rate-limiting/LICENSE:189]: line

**Review by Reecepbcups — APPROVED** [review]
Just the 1 line in the LICENSE

File: modules/rate-limiting/LICENSE (hunk 1 above)
Comment (sampocs) [reply modules/rate-limiting/LICENSE:189]: lol good eye!

**Comment by Reecepbcups** [comment]
You can merge when ready :saluting_face: 

Then when you tag, tag as `modules/rate-limiting/v7.0.0` off of release/v7

image

**Pull request merged**